src/renderer/prefComponents/editor/index.vue

```diff
@@ -34,83 +34,83 @@
+      description="The font used in code blocks."
       :onlyMonospace="true"
       :value="codeFontFamily"
       :onChange="value => onSelectChange('codeFontFamily', value)"
     ></font-text-box>
     <!-- FIXME: Disabled due to #1648. -->
     <bool
       v-show="false"
-      description="Whether to show the code block line numbers."
+      description="Whether to show line numbers in code blocks."
       :bool="codeBlockLineNumbers"
       :onChange="value => onSelectChange('codeBlockLineNumbers', value)"
     ></bool>
     <bool
-      description="Trim the beginning and ending empty lines in code block when open markdown."
+      description="Trim any empty lines at the beginning and end of code blocks when you open a markdown file."
       :bool="trimUnnecessaryCodeBlockEmptyLines"
       :onChange="value => onSelectChange('trimUnnecessaryCodeBlockEmptyLines', value)"
     ></bool>
     <separator></separator>
     <bool
-      description="Automatically brackets when editing."
+      description="Automatically close brackets when writing."
       :bool="autoPairBracket"
       :onChange="value => onSelectChange('autoPairBracket', value)"
     ></bool>
     <bool
-      description="Autocomplete markdown syntax"
+      description="Automatically complete markdown syntax."
       :bool="autoPairMarkdownSyntax"
       :onChange="value => onSelectChange('autoPairMarkdownSyntax', value)"
     ></bool>
     <bool
-      description="Automatic completion of quotes"
+      description="Automatically complete quotes."
       :bool="autoPairQuote"
       :onChange="value => onSelectChange('autoPairQuote', value)"
     ></bool>
     <separator></separator>
     <cur-select
-      description="The default end of line character. If you select default, the ending will be selected according to your system intelligence."
+      description="The newline character used at the end of each line. The "default" option selects your operating system's default newline character."
       :value="endOfLine"
       :options="endOfLineOptions"
       :onChange="value => onSelectChange('endOfLine', value)"
     ></cur-select>
     <separator></separator>
     <cur-select
-      description="The default file encoding."
+      description="The default encoder for files."
       :value="defaultEncoding"
       :options="defaultEncodingOptions"
       :onChange="value => onSelectChange('defaultEncoding', value)"
     ></cur-select>
     <bool
-      description="Try to automatically guess the file encoding when opening files."
+      description="Automatically guess the file's encoding when opening files."
       :bool="autoGuessEncoding"
       :onChange="value => onSelectChange('autoGuessEncoding', value)"
     ></bool>
     <cur-select
-      description="Whether a single trailing newline should be ensured or trailing newlines should be removed."
+      description="Whether trailing newlines should be removed."
```

Comment:
I think a good improvement would be >The trailing newlines options.< or similar.